src/main/java/com/komandux/swag/Controller/OrderAndRefundController.java

```diff
@@ -0,0 +1,168 @@
+import com.komandux.swag.Model.RefundTicket;
+import com.komandux.swag.Model.RefundType;
+import com.komandux.swag.Model.State;
+
+import io.swagger.annotations.ApiOperation;
+
+
+@RestController
+@RequestMapping(name="Order ops")
+public class OrderAndRefundController {
+
+	public List<Product> products = new ArrayList<Product>();
+	
+	{
+		products.add(new Product(0, "barcode", "product1", new BigDecimal("0.1")));
+		products.add(new Product(1, "barcode", "product2", new BigDecimal("0.1")));
+		products.add(new Product(2, "barcode", "product3", new BigDecimal("0.1")));
+		products.add(new Product(3, "barcode", "product4", new BigDecimal("0.1")));
+	}
+	
+	List<Order> orders = new ArrayList<Order>();
+	
+	{
+		orders.add(new Order(0, new Date(), State.Approved, DeliveryMethod.DeliveryToHome, new Date(), PaymentMethod.BankTransfer, "details", "comment"));
+		orders.add(new Order(1, new Date(), State.Shipping, DeliveryMethod.DeliveryToPostMachine, new Date(), PaymentMethod.BankTransfer, "details", "comment"));
+		orders.add(new Order(2, new Date(), State.Pending, DeliveryMethod.InPersonPickup, new Date(), PaymentMethod.BankTransfer, "details", "comment"));
+		orders.add(new Order(3, new Date(), State.Delivered, DeliveryMethod.DeliveryToHome, new Date(), PaymentMethod.BankTransfer, "details", "comment"));
+	}
+	
+	List<OrderItem> orderItemList = new ArrayList<OrderItem>();
+	{
+		orderItemList.add(new OrderItem(1, 1));
+		orderItemList.add(new OrderItem(1, 2));
+		orderItemList.add(new OrderItem(2, 1));
+		orderItemList.add(new OrderItem(2, 1));
+	}
+	
+	List<RefundTicket> refunds = new ArrayList<RefundTicket>();
+	{
+		refunds.add(new RefundTicket(0, 0, new Date(), false, "reason", null));
+		refunds.add(new RefundTicket(1, 1, new Date(), false, "reason", null));
+		refunds.add(new RefundTicket(2, 2, new Date(), false, "reason", null));
+		refunds.add(new RefundTicket(3, 3, new Date(), false, "reason", null));
+		refunds.add(new RefundTicket(4, 4, new Date(), false, "reason", null));
+	}
+	
+	@ApiOperation(value = "Create new order", tags="Order")
+	@RequestMapping(value = "/order", method = RequestMethod.POST)
+	public Order createOrder(@RequestBody Order order) {
+		
+		List<Integer> listIds = orders.stream().map(u -> u.getOrderId()).collect(Collectors.toList());
+		int newId= listIds.stream().max(Integer::compareTo).get() + 1;
+		
+		orders.add(new Order(newId, new Date(), order.getState(), order.getDeliveryMethod(), order.getPurchaseDate(), order.getPaymentMethod(), order.getTransactionDetails(), order.getAdditionalComments()));
+		
+		return orders.get(orders.size()-1);		
+	}
+	
+	@ApiOperation(value = "delete order by ID", tags="Order")
+	@RequestMapping(value = "/OrderDeleteById/{id}", method = RequestMethod.DELETE)
+	public ResponseEntity<?> deleteProductById(@PathVariable("id") int id) {
+		
+		int index=-1;
+		
+		for(int i=0; i<orders.size(); i++) {
+			if(orders.get(i).getOrderId()==id) {
+				index=i;
+			}
+		}
+		
+		if(index>-1) {
```

Comment:
Šitą man atrodo galėjai padaryt pačiame if'e, kad jei randi tai ištrini ir return 200; o po viso for parašyt return 404;